dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-duplicate-host-different-service.eskip

```diff
@@ -0,0 +1,10 @@
+kube_foo__qux______: *
+	-> <roundRobin, "http://10.2.9.103:2134", "http://10.2.9.104:2134">;
+
+kube_foo__qux__www_example_org_____bar:
+	Host("^(www[.]example[.]org[.]?(:[0-9]+)?)$") && PathRegexp("^/")
+	-> <roundRobin, "http://10.2.9.103:8080", "http://10.2.9.104:8080">;
+
+kube_foo__qux__www_example_org_____bar2:
+	Host("^(www[.]example[.]org[.]?(:[0-9]+)?)$") && PathRegexp("^/")
+	-> <roundRobin, "http://10.2.9.105:8081", "http://10.2.9.106:8081">;
```

Comment:
So route id contains service name but not port name/number.
So if we have an ingress with two rules using the same service with two different ports we would imit the second route?...